ManZafAPI/Controllers/LeaveController.cs

```diff
@@ -30,8 +30,15 @@ public async Task<ActionResult<List<LeaveWithoutWorkerDto>>> GetAvailableLeavesF
         public async Task<ActionResult<LeaveDto>> UpdateAvailableLeaveForWorker(LeaveForUpdate leave, int workerId)
         {
             leave.WorkerId = workerId;
-            var updatedLeave = await leaveService.UpdateAvailableLeavesForWorker(mapper.Map<Leave>(leave));
+            var updatedLeave = await leaveService.UpdateAvailableLeaveForWorker(mapper.Map<Leave>(leave));
             return Ok(mapper.Map<LeaveDto>(updatedLeave));
         }
+        [HttpPut("{workerId}/adddays")]
```

Comment:
add-days
https://blog.dreamfactory.com/best-practices-for-naming-rest-api-endpoints/